_episodes/67.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: episode
+title: "67: Extra speed with Exasol and Trino"
+date: 2024-10-21
+tags: trino exasol query performance olap
+youtube_id: "qH61QQJKYPA"
+wistia_id: "update date to 24"
+sections:
+- time: 0:00
+  title: Introduction
+- time: 0:53
+  title: Trino releases 461, 462, and 463
+- time: 9:30
+  title: Guests introductions
+- time: 1:00:42
+  title: Upcoming events and rounding out
+introduction: |
+  Manfred is joined by Thomas Bestfleisch from the Exasol team to talk about Exasol,
+  the new Trino Exasol connector, and some related use cases.
+---
+
+## Host
+
+* [Manfred Moser](https://www.linkedin.com/in/manfredmoser), Director of Trino
+  Community Leadership at [Starburst]({{site.url}}/users.html#starburst),
+  ([@simpligility](https://x.com/simpligility))
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Firebolt](https://www.firebolt.io/)
+  
+## Guests
+
+* [Thomas Bestfleisch](https://www.linkedin.com/in/thomas-bestfleisch/), 
+  Senior Product Manager at [Exasol](https://www.exasol.com/)
+
+## Releases and news
+
+Follow are some highlights of the recent Trino releases:
+
+[Trino 461](https://trino.io/docs/current/release/release-461.html)
+
+* Add support for the `add_files` and `add_files_from_table` procedures in the
+  Iceberg connector.
+
+[Trino 462](https://trino.io/docs/current/release/release-462.html)
+
+* Add support for read operations when using the Unity catalog as Iceberg REST
+  catalog in the Iceberg connector.
+* Improve performance and memory usage when decoding data in the CLI.
+
+[Trino 463](https://trino.io/docs/current/release/release-463.html)
+
+* Enable HTTP/2 for internal communication by default.
+* Add `timezone()` functions.
+* Include table functions with `SHOW FUNCTIONS` output.
+* Add support for writing change data feed when deletion vector is enabled to
+  the Delta Lake connector.
+
+[Trino 464](https://trino.io/docs/current/release/release-464.html)
+
+* Require JDK 23 to run Trino.
+* Add the Faker connector.
+* Add the Vertica connector.
+* Remove the Accumulo connector.
+
+As usual, numerous performance improvements, bug fixes, and other features were
+added as well.
+
+* Trino maintainer call - great sync with some exciting news coming to the community soon. 
+* Trino contributor call - [recording and minutes available now](https://github.com/trinodb/trino/wiki/Contributor-meetings#trino-contributor-call-24-oct-2024). 
+* Trino K8s operator meeting - minutes coming soon.
+* Trino Summit call for speakers closed - stay tuned for announcements.
```

Comment:
Maybe let's highlight that Trino Summit is coming soon and people should register, so there's a clearer call to action.